.gitignore

```diff
@@ -1,2 +1,5 @@
 _kubevirtci/
 build/
+.idea/
+bin/
```

Comment:
where this is used?
.vscode needed as well i think

Reply:
Regarding the `bin`, I"m not sure maybe by the operator-sdk.
I have `ls -l bin/
controller-gen  k8s/            setup-envtest`
But I think makes sesne anyway to have it in the .gitignore.
Will add .vscode